docs/feature-gates.md

```diff
@@ -96,4 +97,38 @@ to a flow collector. Refer to this [document](network-flow-visibility.md) for mo
 #### Requirements for this Feature
 
 This feature is currently only supported for Nodes running Linux.
-Windows support will be added in the future.
\ No newline at end of file
+Windows support will be added in the future.
+
+### NetworkPolicyStats
+
+`NetworkPolicyStats` enables collecting NetworkPolicy metrics from antrea-agents
+and exposing them through Antrea Stats API, which can be accessed by kubectl get
+commands, e.g. `kubectl get networkpolicystats`. The statistical data includes
+total number of sessions, packets, and bytes enforced by a NetworkPolicy. It is
+collected offline so it may have a delay up to 1 minute to reflect in response
```

Comment:
I'm thinking "asynchronously" is more accurate than "offline" here, but I may be wrong

Reply:
done